corehq/apps/cloudcare/static/cloudcare/js/form_entry/utils.js

```diff
@@ -103,6 +103,12 @@ hqDefine("cloudcare/js/form_entry/utils", function () {
                 $("#" + divId + "-sr").html("<p>" + currentOption + "</p>");
             }
         }, 200));
+
+        geocoder.on('results', (items) => {
+            if (items && items.features) {
+                $("#" + divId + "-sr").html("<p>" + items.features[0].place_name + "</p>");
+            }
+        });
```

Comment:
Oh wow! please add a comment for this as well

Reply:
added comment for this. [f43b3d7](https://github.com/dimagi/commcare-hq/pull/32637/commits/f43b3d742c601aafd2ff7b7a37c89e2d11ce55c1)